### DISCUSSION TITLE
UNO and PRISM not building

### DISCUSSION THREAD

**Discussion by apovetkin** [unoplatform/uno#7920] [Q&A] [answered]
Hi guys,

I tried to build Uno and PRISM and have the error. Can you please watch the video [https://youtu.be/-p1JBqrGqvg](url)

Thanks,
Alexander

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. 

The prism template cannot work until a newer version of the Prism library gets published to support Uno 4.0. It's been fixed by https://github.com/PrismLibrary/Prism/pull/2628, and pending an update from the Prism lib developers, see the PR comments for information.
